sources/Application/Views/InstrumentView.cpp

```diff
@@ -1020,3 +1049,12 @@ void InstrumentView::AnimationUpdate() {
   drawBattery(props);
   w_.Flush();
 };
+
+// Redraw all UI fields to reflect updated variable values
+void InstrumentView::redrawAllFields() {
```

Comment:
This can be removed because I added it in my implementation but its not needed anymore.